container-images/default.Containerfile

```diff
@@ -32,9 +27,9 @@ RUN dnf install ${DNF_DEFAULT_OPTS} \
     python3-setuptools \
     python3-wheel
 
-RUN pip3.12 install -U poetry && poetry install
+RUN curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/usr/local/bin" sh && uv sync
```

Comment:
I wonder if it would make more sense to do a `uv export` instead of installing it in the container just to sync the deps. Maybe the workflow would get a bit more complicated, as I believe we would export the requirements.txt before we start the container build. 

What do you think? Not strong preference for me, just wondering if it is worth it or not.

Reply:
Where in the build step would you run `uv export`? The way this Containerfile
builds right now, `uv` is installed into the build container and not the final
container. Are you thinking about keeping a `requirements.txt` committed in the
repository along side the `uv.lock`?